packages/router/src/adapters/subgraph/graphqlsdk.ts

```diff
@@ -804,126 +988,120 @@ export const GetReceiverTransactionsDocument = gql`
+    GetReceiverTransactions(
+      variables: GetReceiverTransactionsQueryVariables,
+      requestHeaders?: Dom.RequestInit["headers"],
+    ): Promise<GetReceiverTransactionsQuery> {
+      return withWrapper(
+        (wrappedRequestHeaders) =>
+          client.request<GetReceiverTransactionsQuery>(GetReceiverTransactionsDocument, variables, {
+            ...requestHeaders,
+            ...wrappedRequestHeaders,
+          }),
+        "GetReceiverTransactions",
+      );
     },
-    GetTransaction(variables: GetTransactionQueryVariables, requestHeaders?: Dom.RequestInit["headers"]): Promise<GetTransactionQuery> {
-      return withWrapper((wrappedRequestHeaders) => client.request<GetTransactionQuery>(GetTransactionDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), 'GetTransaction');
+    GetTransaction(
+      variables: GetTransactionQueryVariables,
+      requestHeaders?: Dom.RequestInit["headers"],
+    ): Promise<GetTransactionQuery> {
+      return withWrapper(
+        (wrappedRequestHeaders) =>
+          client.request<GetTransactionQuery>(GetTransactionDocument, variables, {
+            ...requestHeaders,
+            ...wrappedRequestHeaders,
+          }),
+        "GetTransaction",
+      );
     },
-    GetTransactions(variables?: GetTransactionsQueryVariables, requestHeaders?: Dom.RequestInit["headers"]): Promise<GetTransactionsQuery> {
-      return withWrapper((wrappedRequestHeaders) => client.request<GetTransactionsQuery>(GetTransactionsDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), 'GetTransactions');
+    GetTransactions(
+      variables?: GetTransactionsQueryVariables,
+      requestHeaders?: Dom.RequestInit["headers"],
+    ): Promise<GetTransactionsQuery> {
+      return withWrapper(
+        (wrappedRequestHeaders) =>
+          client.request<GetTransactionsQuery>(GetTransactionsDocument, variables, {
+            ...requestHeaders,
+            ...wrappedRequestHeaders,
+          }),
+        "GetTransactions",
+      );
     },
-    GetAssetBalance(variables: GetAssetBalanceQueryVariables, requestHeaders?: Dom.RequestInit["headers"]): Promise<GetAssetBalanceQuery> {
-      return withWrapper((wrappedRequestHeaders) => client.request<GetAssetBalanceQuery>(GetAssetBalanceDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), 'GetAssetBalance');
+    GetAssetBalance(
+      variables: GetAssetBalanceQueryVariables,
+      requestHeaders?: Dom.RequestInit["headers"],
+    ): Promise<GetAssetBalanceQuery> {
+      return withWrapper(
+        (wrappedRequestHeaders) =>
+          client.request<GetAssetBalanceQuery>(GetAssetBalanceDocument, variables, {
+            ...requestHeaders,
+            ...wrappedRequestHeaders,
+          }),
+        "GetAssetBalance",
+      );
+    },
+    UpdateSyncStatus(
```

Comment:
MARK: Only real change in this file is here: `UpdateSyncStatus`; all other changes are eslint/Prettier formatting.